src/Titon/Route/Router.hh

```diff
@@ -637,7 +636,7 @@ class Router implements Subject {
             }
 
             /** @var \Titon\Route\Route $newRoute */
-            $newRoute = Registry::factory($class, [$newPath, static::buildAction($newAction)], false);
+            $newRoute = Depository::getInstance()->make($class, $newPath, static::buildAction($newAction));
```

Comment:
Fixed:

Right. I've reverted that line. It wasn't a component using the FactoryAware trait anyway. May have been a bad search on my part thinking it was using FactoryAware